lib/types/json.js

```diff
@@ -77,10 +77,13 @@ function json(options) {
 
     if (strict) {
       var first = firstchar(body)
-
+      var message = 'Unexpected token ' + first
       if (first !== '{' && first !== '[') {
+        if (process.version[1] >= 6) {
+          message += ' in JSON at position 1'
```

Comment:
No, it throws based on the first non-ignored character. You can look at where that variable is actually coming from.